data/migrations/20201109081432_create-watchlist.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+  return knex.schema
+    .createTable('watchlist', (list) => {
+      list.increments();
+      list.string('list_title', 50).unique().notNullable();
+      list.string('profile_id').unsigned().references('id').inTable('profiles');
+      list
```

Comment:
It looks like this is creating a 1-1 relationship between lists and projects. So a list will only be able to contain a single bridge. We need a 1-many relationship between the list and bridges.